src/main/java/com/github/tomakehurst/wiremock/http/ssl/CertificateAuthority.java

```diff
@@ -37,9 +38,7 @@ public class CertificateAuthority {
 
   public CertificateAuthority(X509Certificate[] certificateChain, PrivateKey key) {
     this.certificateChain = requireNonNull(certificateChain);
-    if (certificateChain.length == 0) {
-      throw new IllegalArgumentException("Chain must have entries");
-    }
+    checkParameter(certificateChain.length == 0, "Chain must have entries");
```

Comment:
Thank you for updating this.  Sorry, I should have checked this in my last review - can you check that the condition is still valid now that you have moved over to using one of the `check` methods.  It seems to be failing tests so I have a feeling it now needs to be `certificateChain.length != 0`.  Take a look at the `checkParameter` method and see what you think.  If you run the test first you will see the failing ones and then run them after the change and hopefully they will be fixed.